piff/gsobject_model.py

```diff
@@ -363,11 +302,13 @@ class Gaussian(GSObjectModel):
                         position is fixed at input value and the fitted PSF may be off-center.
                         [default: True]
     :param include_pixel: Include integration over pixel when drawing?  [default: True]
+    :param scipy_kwargs: Optional kwargs to pass to scipy.optimize.least_squares [default: None]
     :param logger:      A logger object for logging debug info. [default: None]
     """
-    def __init__(self, fastfit=False, centered=True, include_pixel=True, logger=None):
+    def __init__(self, fastfit=False, centered=True, include_pixel=True,
```

Comment:
To be completely honest, it was easier to test some failure modes of the scipy call when I could set e.g. max_nfev=10 so that I could make the scipy.optimize call fail more easily.

But it occurred to me that this could be a useful feature that users might want.  For marginal objects, the default scipy call takes rather a while to run.  Setting the options to something faster that might fail more often (and then these marginal stars would just be excluded) could be useful when you know you have many more stars than you need and you don't want to iterate forever for the few tough nuts.

As for why here, the user doesn't have direct access to the fit() call.  That's usually called from within a larger process of PSF.fit().  When running via the config, the only real place we have to provide extra options for the user to set is at the constructor.